.github/workflows/scan.yml

```diff
@@ -0,0 +1,60 @@
+name: Scan
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+      - "rc"
+      - "hotfix-rc"
```

Comment:
Are we not running full checks on `main`?

Reply:
See https://github.com/bitwarden/server/pull/3832/files#diff-246cd0c2f7db532638dd80a92ac011f49b3d26038983a4c0169ea8f8a5c39280R28 -- this will run full checks outside PRs.